bin/coveralls

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env php
+<?php
+
+if (file_exists(__DIR__ . '/../vendor/autoload.php')) {
+    require __DIR__ . '/../vendor/autoload.php';
+} elseif (file_exists(__DIR__ . '/../../autoload.php')) {
+    require __DIR__ . '/../../autoload.php';
+} else {
+    die(
```

Comment:
If it is called by the phar file, do we still need to require autoload file? If the phar file includes all the deps, there is no need to require them, right?

Is there a way to detect if it's called from within a phar file? If so, can we skip this section in that case?